src/gatsby-node.js

```diff
@@ -68,7 +69,6 @@ exports.sourceNodes = (
       {
         contype: '',
         contypeother: '',
-        journal_name: '',
```

Comment:
I don't think we should remove this yet, because `master` of faculty-profiles mentions this key on line 133 of `gatsby_fp/src/components/Layout.js`